### PR TITLE
Vl gallery layout

### PR DESCRIPTION
**Pull request by vinhle000**
Added layout for thumbnails and main photo for the image gallery.
Implemented functionality for the selected thumbnail to become the main photo presented in the Overview widget.

**Ticket:**
https://trello.com/c/zYbulrmP

**Behavior:**
ImageGallery_ThumbnailSelected_v2

### CODE REVIEW

File: src/components/overview/ImageGallery.jsx [hunk 1]
```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import React, {useState} from 'react';
 import styles from './overview.module.css'
+import json from './styleSample.json';
 
 
 const ImageGallery = (props) => {
 
+  var nums = Array.from(Array(7).keys()) //Used for testing flex box children size
+  // console.log(nums)
```
Comment (merosenlund) [inline src/components/overview/ImageGallery.jsx:9]: Will you still be needing this console log?

File: src/components/overview/overview.module.css [hunk 2]
```diff
@@ -175,4 +175,76 @@
...
+  height: 680px;
+  width: 550px;
+  margin: 10px;
+  border: 1px solid purple;
+}
+
+.mainPhoto{
+  height: 100%;
+  width: auto;
+  justify-content: ;
```
Comment (merosenlund) [inline src/components/overview/overview.module.css:249]: Is this not supposed to have a value assigned?

File: src/components/overview/ImageGallery.jsx [hunk 3]
```diff
@@ -1,11 +1,58 @@
...
+
+  /*Gallery would need
```
Comment (merosenlund) [inline src/components/overview/ImageGallery.jsx:11]: This looks like a comment for referencing while developing? Is it still needed?

File: src/components/overview/ImageGallery.jsx [hunk 4]
```diff
@@ -1,11 +1,58 @@
...
+   -state of style index,
+    -style information,
+    chosen thumbnail index
+    */
+
+
+   var photos = json.results[0].photos;
+  // const [selectedStyle, setStyle] = useState() //Place holder for use of context to change style
```
Comment (merosenlund) [inline src/components/overview/ImageGallery.jsx:19]: This line will no longer be needed when you start using ProductContext?

**Review by merosenlund — COMMENTED** [review]
Other than some (maybe?) unnecessary comments sprinkled around this looks pretty clean and good. 

Also there is that empty justify-content css at the end that I'm not sure about. This is the only potential thing that I could see *needing* to be changed before merging.  Let me know.

File: src/components/overview/ImageGallery.jsx (hunk 4 above)
Comment (vinhle000) [reply src/components/overview/ImageGallery.jsx]: Place holder for next step of implementation. But no longer needed. Removed comment.

File: src/components/overview/overview.module.css (hunk 2 above)
Comment (vinhle000) [reply src/components/overview/overview.module.css]: Removed extra unassigned property. Was used for testing flexbox.

File: src/components/overview/ImageGallery.jsx (hunk 4 above)
Comment (vinhle000) [reply src/components/overview/ImageGallery.jsx]: Removed comment.

File: src/components/overview/ImageGallery.jsx (hunk 1 above)
Comment (vinhle000) [reply src/components/overview/ImageGallery.jsx]: Removed comment.

**Review by merosenlund — APPROVED** [review]

**Pull request merged**